core/src/main/java/org/jivesoftware/sparkimpl/certificates/CertificateController.java

```diff
@@ -41,17 +38,24 @@
  */
 
 public class CertificateController extends CertManager {
-	public final static File TRUSTED = new File(Spark.getSparkUserHome() + File.separator + "security" + File.separator + "truststore");
-	public final static File BLACKLIST = new File(Spark.getSparkUserHome() + File.separator + "security" + File.separator + "blacklist");
-	public final static File EXCEPTIONS = new File(Spark.getSparkUserHome() + File.separator + "security" + File.separator + "exceptions");
-	
-	
-	private KeyStore trustStore, blackListStore, exceptionsStore;
+	public final static File TRUSTED =            new File(Spark.getSparkUserHome() + File.separator + "security" + File.separator + "truststore");
```

Comment:
Please add in javadoc exactly what the purpose of every store is - so that Devs that look at this in the future understand how things are combined, and why.